src/SignalR/common/SignalR.Common/src/PublicAPI.Unshipped.txt

```diff
@@ -1,4 +1,5 @@
 #nullable enable
+Microsoft.AspNetCore.SignalR.IInvocationBinder.GetTarget(System.ReadOnlySpan<byte> utf8Bytes) -> string?
```

Comment:
@dougbu how do I fix this?

Reply:
According to <https://github.com/dotnet/roslyn-analyzers/blob/main/src/PublicApiAnalyzers/PublicApiAnalyzers.Help.md#conditional-api-differences>, you need to add PublicApi/net7.0/PublicApi.Shipped.txt and &hellip;/PublicApi.Unshipped.txt files as well as a reference to them in the project file:

``` msbuild
  <ItemGroup>
    <AdditionalFiles Include="PublicAPI/$(DefaultNetCoreTargetFramework)/PublicAPI.Shipped.txt" />
    <AdditionalFiles Include="PublicAPI/$(DefaultNetCoreTargetFramework)/PublicAPI.Unshipped.txt" />
  </ItemGroup>
```

I think the naming scheme is a bit verbose in those docs. Maybe PublicApi.net7.0.Shipped.txt and &hellip;Unshipped.txt right in the project directory❔

Could also use `netcoreapp` in the filenames to avoid renames when we next update that TFM.

Reply:
Oh, I'm not sure if the `<ItemGroup/>` needs to be conditional if the files don't have `net7.0` in their paths. Might need to check w/ the roslyn-analyzer people or in their code.